packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -109,7 +109,7 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.lookback,
       config.twapLength,
       config.poolDecimals,
-      config.decimals // This defaults to 18 unless supplied by user
+      config.priceFeedDecimals // This defaults to 18 unless supplied by user
```

Comment:
`config.decimals` doesn't actually exist when the config is constructed by `DefaultPriceFeedConfigs`, so I renamed to `priceFeedDecimals` which `CryptoWatchPriceFeed` uses.